zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/OperationReferenceTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.client.command;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.it.util.ZeebeResourcesHelper;
+import io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.camunda.zeebe.test.util.junit.AutoCloseResources.AutoCloseResource;
+import io.camunda.zeebe.test.util.record.ProcessInstanceRecordStream;
+import io.camunda.zeebe.test.util.record.RecordingExporter;
+import java.time.Duration;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class OperationReferenceTest {
```

Comment:
🔧 Please rename to `OperationReferenceIT` as it should be run with the failsafe instead of surefire plugin.

Reply:
If you look at other test classes in the same package around this one, they use the same naming convention I used here. And they perform similar checks. I don't won't this to be the odd one, even if the rest is named incorrectly, I'd stick to the same mistake until we fix all.

Reply:
👍 I see now that it really doesn't matter yet:
https://github.com/camunda/camunda/blob/69f2a999f402ef763991765535e86af86df8fe84/zeebe/qa/pom.xml#L43-L50